PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cff.py

```diff
@@ -18,3 +18,25 @@
     patJets
     )
 makePatJets = cms.Sequence(makePatJetsTask)
+
+from RecoBTag.ImpactParameter.pfImpactParameterTagInfos_cfi import * #pfImpactParameterTagInfos
+from RecoBTag.SecondaryVertex.pfSecondaryVertexTagInfos_cfi import * #pfSecondaryVertexTagInfos
+from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderTagInfos_cfi import * #pfInclusiveSecondaryVertexFinderTagInfos
+from RecoBTag.Combined.deepFlavour_cff import * #pfDeepFlavourTask
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+#make a copy to avoid labels and substitution problems
+_makePatJetsCopy = makePatJets.copy()
+
+run2_miniAOD_80XLegacy.toReplaceWith(
+    makePatJets, 
+    cms.Sequence(
```

Comment:
@slava77 I checked cmsDriver and python to not complain about anything, but I did not fully tested it on data because I do not know all the customization needed for the miniAOD on legacy. If you can suggest a workflow and an input file I will test it and, in case, fix everything